.gitlab-ci.yml

```diff
@@ -33,10 +34,11 @@ docker:build:branches:
     - branches
   script:
     - docker login -u ${CI_REGISTRY_USER} -p ${CI_REGISTRY_PASS} ${CI_REGISTRY}
-    - docker build
-      --build-arg VCS_REF=`git rev-parse --short HEAD`
-      --build-arg BUILD_DATE=`date -u +"%Y-%m-%dT%H:%M:%SZ"`
-      --build-arg VERSION=${VERSION}
+    - docker build 
+      --cache-from=${DOCKER_IMAGE}:${CI_COMMIT_REF_NAME/\//-} 
```

Comment:
Use directly `${CI_COMMIT_REF_SLUG}` instead of your replace `${CI_COMMIT_REF_NAME/\//-}`

Reply:
DONE